hermes-tracker-elasticsearch/src/main/java/pl/allegro/tech/hermes/tracker/elasticsearch/consumers/ElasticsearchLogRepository.java

```diff
@@ -77,4 +80,63 @@ protected XContentBuilder notEndedDocument(MessageMetadata message, long timesta
                 .field(CLUSTER, clusterName);
     }
 
+    public static class Builder {
```

Comment:
I wonder if not give more descriptive name for builder 
and extract it for using it also in  ...ava/pl/allegro/tech/hermes/tracker/elasticsearch/frontend/ElasticsearchLogRepository.java with just few more params